test/test_plugin_helper.rb

```diff
@@ -29,7 +29,7 @@ def setup_routes
   end
 
   def setup_templates
-    load File.join(File.dirname(__FILE__), '..', 'db', 'seeds.d', '50-bootdisk_templates.rb')
+    load File.join(File.dirname(__FILE__), '..', '..', 'foreman', 'db', 'seeds.d', '070-provisioning_templates.rb')
```

Comment:
This wont work, as you can see from the failing tests:
```
Error:
ForemanBootdisk::Api::V2::DisksControllerTest::default API version 2#test_0002_path - /api/generic routes to #generic:
LoadError: cannot load such file -- /home/runner/work/foreman_bootdisk/foreman_bootdisk/foreman_bootdisk/test/../../foreman/db/seeds.d/070-provisioning_templates.rb
```

That file must be in this repository

Reply:
Yes yes, I know. I'm still trying to find a solution for this. This file cannot be in this repository since I deleted the templates and moved them to foreman. So I need to find another way, like using fixtures like I did in foreman_discovery.

Reply:
Well it can be still in this repository, but as a text fixture, in `test/static_fixtures/template.rb`